src/Globe/Atmosphere.js

```diff
@@ -96,95 +93,28 @@ function Atmosphere(ellipsoid) {
     };
 
     var uniformsSky = {
-        v3LightPosition: {
-            type: 'v3',
-            value: LIGHTING_POSITION.clone().normalize(),
-        },
-        v3InvWavelength: {
-            type: 'v3',
-            value: new THREE.Vector3(1 / Math.pow(atmosphere.wavelength[0], 4), 1 / Math.pow(atmosphere.wavelength[1], 4), 1 / Math.pow(atmosphere.wavelength[2], 4)),
-        },
-        fCameraHeight: {
-            type: 'f',
-            value: 0.0,
-        },
-        fCameraHeight2: {
-            type: 'f',
-            value: 0.0,
-        },
-        fInnerRadius: {
-            type: 'f',
-            value: atmosphere.innerRadius,
-        },
-        fInnerRadius2: {
-            type: 'f',
-            value: atmosphere.innerRadius * atmosphere.innerRadius,
-        },
-        fOuterRadius: {
-            type: 'f',
-            value: atmosphere.outerRadius,
-        },
-        fOuterRadius2: {
-            type: 'f',
-            value: atmosphere.outerRadius * atmosphere.outerRadius,
-        },
-        fKrESun: {
-            type: 'f',
-            value: atmosphere.Kr * atmosphere.ESun,
-        },
-        fKmESun: {
-            type: 'f',
-            value: atmosphere.Km * atmosphere.ESun,
-        },
-        fKr4PI: {
-            type: 'f',
-            value: atmosphere.Kr * 4.0 * Math.PI,
-        },
-        fKm4PI: {
-            type: 'f',
-            value: atmosphere.Km * 4.0 * Math.PI,
-        },
-        fScale: {
-            type: 'f',
-            value: 1 / (atmosphere.outerRadius - atmosphere.innerRadius),
-        },
-        fScaleDepth: {
-            type: 'f',
-            value: atmosphere.scaleDepth,
-        },
-        fScaleOverScaleDepth: {
-            type: 'f',
-            value: 1 / (atmosphere.outerRadius - atmosphere.innerRadius) / atmosphere.scaleDepth,
-        },
-        g: {
-            type: 'f',
-            value: atmosphere.g,
-        },
-        g2: {
-            type: 'f',
-            value: atmosphere.g * atmosphere.g,
-        },
-        nSamples: {
-            type: 'i',
-            value: 3,
-        },
-        fSamples: {
-            type: 'f',
-            value: 3.0,
-        },
-
-        tDisplacement: {
-            type: 't',
-            value: new THREE.Texture(),
-        },
-        tSkyboxDiffuse: {
-            type: 't',
-            value: new THREE.Texture(),
-        },
-        fNightScale: {
-            type: 'f',
-            value: 1.0,
-        },
+        v3LightPosition: { value: LIGHTING_POSITION.clone().normalize() },
```

Comment:
Nice change, but it would have been easier to review if you had split all the code cleanup in a separate PR.